src/dune_engine/dtemp.ml

```diff
@@ -0,0 +1,24 @@
+open Stdune
+
+let temp_dir = lazy (Temp.create Dir ~prefix:"build" ~suffix:".dune")
```

Comment:
I don't get it. `Temp.create` seems to be using `Filename.get_temp_dir_name`, which, presumably, will still create files in /tmp or $TMPDIR.

Don't we want this to be in the build directory, instead?

Reply:
> Don't we want this to be in the build directory, instead?

That's how I did it initially, but then Jeremie pointed out that writing to `/tmp` can be much faster if it's on ramfs. We decided that it was more conservative not to use `_build` after all.

Reply:
Got it!